docs/3.0rc/develop/task-run-limits.mdx

```diff
@@ -9,23 +9,15 @@ For example, if many tasks across multiple flows are designed to interact with a
 Task run concurrency limits use [task tags](#tags). You can specify an optional concurrency limit as the maximum number of concurrent 
 task runs in a `Running` state for tasks with a given tag.
 
-If a task has multiple tags, it will run only if _all_ tags have available concurrency.
-
-Tags without explicit limits are considered to have unlimited concurrency.
+<Tip>
+Tag-based task concurrency is different from [Global Concurrency Limits](/3.0rc/develop/global-concurrency-limits), though they can be used to achieve similar outcomes. Global Concurrency Limits are a more general way to control concurrency for any python-based operation, whereas tag-based concurrency limits are specific to Prefect tasks.
+</Tip>
 
-{/*
-<!-- vale off -->
-*/}
 
-<Note>
-**0 concurrency limit aborts task runs**
 
-If the concurrency limit is set to 0 for a tag, any attempt to run a task with that tag will abort instead of delay.
-</Note>
+If a task has multiple tags, it will run only if **_all_** tags have available concurrency.
 
-{/*
-<!-- vale on -->
-*/}
+Tags without explicit limits are considered to have unlimited concurrency, and **a concurrency limit of 0 will abort task runs**. That is to say, if the concurrency limit is set to 0 for a tag, any attempt to run a task with that tag will abort instead of delay.
```

Comment:
Potential wording without repetition:
```suggestion
Tags without specified concurrency limits are treated as unlimited. Setting a tag's concurrency limit to 0 causes immediate abortion of any task runs with that tag, rather than delaying them.
```